PFOE-40 Komponent do wyswietlania szczegółów wydarzenia.

HubertMos99 opened the pull request:
Żeby po kliknięciu przycisku 'zapisz sie', dodawalo dane do bazy danych, musze dodać metode do serwisu?

HubertMos99 committed:
PFOE-40 Komponent do wyswietlania szczegółów wydarzenia.
Żeby po kliknięciu przycisku 'zapisz sie', dodawalo dane do bazy danych, musze dodać metode do serwisu?

MCMisha commented:
> Żeby po kliknięciu przycisku 'zapisz sie', dodawalo dane do bazy danych, musze dodać metode do serwisu?

Tak. Jeszcze musisz dodać endpoint na backendzie, żeby zwracał ilość uczestników do wydarzenia.
![image](https://github.com/MCMisha/PFOE/assets/47827537/ff354974-1f2f-4723-9189-544e2eaa0119)

MCMisha commented on frontend/pfoe-front/src/app/events/eventsView/eventsView.component.html:
```diff
@@ -0,0 +1,42 @@
+    <h3 style="margin-bottom: 30px;"><strong>{{currentEvent?.name}}</strong></h3>
+    <p><strong>Data: </strong>{{currentEvent?.date}}</p>
+    <p><strong>Kategoria: </strong>{{currentEvent?.category}}</p>
+    <p><strong>Ilość uczestników: </strong>{{currentEvent?.participant_number}}</p>
```
Musisz zmienić property 'participant_number' na 'participantNumber' w modelu

MCMisha commented on frontend/pfoe-front/src/app/events/eventsView/eventsView.component.html:
```diff
@@ -0,0 +1,42 @@
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Event Details</title>
+  <style>
```
Te styli do 'eventsVies.component.scss'

HubertMos99 committed:
PFOE-40 Komponent do wyswietlania szczegółów wydarzenia.
Żeby po kliknięciu przycisku 'zapisz sie', dodawalo dane do bazy danych, musze dodać metode do serwisu? I musze jeszcze do modeli dodac Participant?

MCMisha committed:
Fix WebApi.csproj.nuget.g.props

Signed-off-by: misha <misha.yakushevich4@gmail.com>

HubertMos99 committed:
PFOE-40 Komponent do wyswietlania szczegółów wydarzenia.

HubertMos99 committed:
Merge branch 'dev' into PFOE-40

# Conflicts:
#	backend/WebApi/WebApi/Controllers/EventController.cs
#	backend/WebApi/WebApi/Controllers/UserController.cs
#	backend/WebApi/WebApi/Repositories/UserRepository.cs
#	backend/WebApi/WebApi/Services/EventService.cs
#	backend/WebApi/WebApi/Services/UserService.cs
#	frontend/pfoe-front/src/app/events/events.component.html
#	frontend/pfoe-front/src/app/events/events.component.ts
#	frontend/pfoe-front/src/app/events/events.module.ts
#	frontend/pfoe-front/src/app/main/main-routing.module.ts
#	frontend/pfoe-front/src/app/main/main.component.html
#	frontend/pfoe-front/src/app/services/event.service.ts

HubertMos99 committed:
PFOE-40 Komponent do wyswietlania szczegółów wydarzenia.

MCMisha committed:
PFOE-40 Poprawki
